### DISCUSSION TITLE
挂载阿里云盘的时候能否同时挂载多个想要挂载的文件夹

### DISCUSSION THREAD

**Discussion by Sikongyi** [AlistGo/alist#3900] [General]
![image](https://user-images.githubusercontent.com/125432341/226249747-e0b3c734-9ef4-4469-a473-27e46064d1cd.png)
如果可以选择挂载多个文件夹，根文件ID这里需要怎么填写呢？

**Comment by Animism001** [comment]
意思是要挂载同一个阿里云盘的不同文件夹？打开对应的阿里云盘目录，将网址的最后一串字符填到根文件夹ID，例如http://123/456/root，将root填进去。具体可以看alist帮助文档中的添加阿里云盘的说明

**Reply by Sikongyi** [reply]
那要同时挂载这个盘里的多个文件夹，比如这两个根文件ID为root1,root2.要如何填写呢。

**Comment by Animism001** [comment]
不能一次性挂载两个文件夹的，你只能添加对应数量的存储。

**Comment by myfreewiki** [comment]
新建一个文件夹，把需要显示的文件、文件夹放里面，然后挂载这个文件夹。
